code/_onclick/other_mobs.dm

```diff
@@ -24,9 +24,6 @@
 	var/obj/item/clothing/gloves/G = gloves // not typecast specifically enough in defines
 	if(proximity && istype(G) && G.Touch(A,1))
 		return
-	//This signal is needed to prevent gloves of the north star + hulk.
-	if(SEND_SIGNAL(src, COMSIG_HUMAN_EARLY_UNARMED_ATTACK, A, proximity) & COMPONENT_CANCEL_ATTACK_CHAIN)
```

Comment:
Keep this in, there's still plenty of stuff that uses COMSIG_HUMAN_EARLY_UNARMED_ATTACK, and we don't want whatever bug this was fixing cropping back up.

Reply:
I put that back in. I assumed it was specifically hulk related, but I see why that's important to keep in.